x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -268,10 +289,16 @@ export const CommandInput = memo<CommandInputProps>(({ prompt = '', focusRef, ..
                 <div data-test-subj={getTestId('cmdInput-rightOfCursor')}>{rightOfCursor.text}</div>
               </EuiFlexItem>
             </EuiFlexGroup>
-
             <InputPlaceholder />
           </EuiFlexItem>
-          <EuiFlexItem grow={false} />
+          <EuiFlexItem grow={false}>
```

Comment:
can we remove this change and handle it in a separate PR. There is more on the behaviour of this icon that just displaying it.  From yesterday's grooming during the working group team, I understood that:

1. it will be a button
2. button will be disabled until any text is entered
3. when enabled, it color is blue
4. click on it will `{ENTER}` the current command in the input area

@kevinlog  - is there a separate issue/task for this?

Reply:
@paul-tavares I will make one for it.  And yes, I think we should do it in a follow up PR since that functionality wasn't in the original ask.